ecowitt2mqtt/runtime.py

```diff
@@ -46,6 +46,7 @@ class Runtime:  # pylint: disable=too-many-instance-attributes
     def __init__(self, ecowitt: Ecowitt) -> None:
         """Initialize."""
         self.ecowitt = ecowitt
+        self.mqtt_connected = False
```

Comment:
Let's make this a private attribute:

```suggestion
        self._mqtt_connected = False
```